File: 06_Prvni_aplikace/PrvniAplikaceJava/app/src/main/java/cz/programujemeproandroid/prvniaplikacejava/MainActivity.java
package cz.programujemeproandroid.prvniaplikacejava;

import androidx.appcompat.app.AppCompatActivity;

import android.app.AlertDialog;
import android.os.Bundle;
import android.view.View;

import java.util.ArrayList;
import java.util.List;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void kliknutiNaTlacitko(View sender) {
        AlertDialog.Builder builder = new AlertDialog.Builder(MainActivity.this);
        builder.setCancelable(true);
        builder.setTitle("Poznámka");
        builder.setMessage("Android je zde!");
        builder.setPositiveButton("Zavřít", null);

        builder.create().show();

        return;
    }
}

File: 06_Prvni_aplikace/PrvniAplikaceKotlin/app/src/main/java/cz/programujemeproandroid/prvniaplikacekotlin/MainActivity.kt
package cz.programujemeproandroid.prvniaplikacekotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AlertDialog

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun kliknutiNaTlacitko(sender: View) {
        val builder = AlertDialog.Builder(this@MainActivity)
        builder.setCancelable(true)
        builder.setTitle("Poznámka")
        builder.setMessage("Android je zde!")
        builder.setPositiveButton("Zavřít", null)
        builder.create().show()

        return
    }
}
